lib/Doctrine/Inflector/Rules/English/Inflectible.php

```diff
@@ -18,6 +18,7 @@ public static function getSingular() : iterable
     {
         yield new Transformation(new Pattern('(s)tatuses$'), '\1\2tatus');
         yield new Transformation(new Pattern('(s)tatus$'), '\1\2tatus');
+        yield new Transformation(new Pattern('campus$'), 'campus');
```

Comment:
This should be case sensitive like other patterns around it.

Reply:
Fixed.